tfhe/src/integer/bigint/mod.rs

```diff
@@ -6,6 +6,11 @@ pub mod static_unsigned;
 pub mod u256;
 pub mod u512;
 
+pub use static_unsigned::StaticUnsignedBigInt;
+
+pub type U2048 = static_unsigned::StaticUnsignedBigInt<32>;
+pub type U4096 = static_unsigned::StaticUnsignedBigInt<64>;
```

Comment:
question to maybe use bits instead of 64 bits as a reference for the number of blocks ?

and maybe restrain to multiple of 64 ? unsure but this looks error prone

Reply:
It's not possible, rust const generic are too limited we cannot do const operations with them

Reply:
yeah, I seemed to recall that thanks